client/command_data/skillorder.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @fileoverview Command Data & Constants Related to the /about command
+ * These constants are used to dynamically generate a slash command on discord.
+ * This File also contains the constants values used exclusively in the /about command.
+ * @author Ai0796
+ */
+
+module.exports = {
+    'INFO': {
+        'name': 'skillorder',
+        'utilization': '/skillorder',
+        'description': 'Show the current T100 leaderboard',
+        'ephemeral': false,
+        'params': [
+            {
+                'type': 'integer',
+                'name': 'song',
+                'required': false,
+                'description': 'Song Name',
+                'choices': [
+                    ["Melt", 47],
```

Comment:
Hi Ai0796,
I like how you're using the command parser to generate this command.
I just wanted to give the heads up that the 'choices' field in discord slash commands are very limited. If you are considering increasing this number in the future, you might hit a point where you can't add anymore choices to the command.

You might want to try and create a `/skillorder` command which opens up a scrollable? (you can decide the design implementation here) selection of buttons containing songs. Basically like the /skillorder command opens up a selection menu, rather then directly supplying the output.

Reply:
Changes from Melt and Envy to a larger list of most popular tiering songs. Need feedback if people care about skill orders in other songs due to limited effect.

Reply:
Looks good to me